datacapture/src/main/java/com/google/android/fhir/datacapture/QuestionnaireViewModel.kt

```diff
@@ -99,6 +100,16 @@ internal class QuestionnaireViewModel(application: Application, state: SavedStat
     }
   }
 
+  /** Flag to open fragment in data-collection or review-mode */
+  internal val initialReviewMode: Boolean
```

Comment:
can this be private?

```suggestion
  private val initialReviewMode: Boolean
```

Reply:
Done